### PR TITLE
Add capability to declare specific attribute

### PR DESCRIPTION
**Pull request by VictorRodriguez**
Try to provide a common fix for https://github.com/clearlinux/make-fmv-patch/issues/8

Signed-off-by: VictorRodriguez <vm.rod25@gmail.com>

### CODE REVIEW

**Comment by VictorRodriguez** [comment]
@zerodefect could you please help me to take a look at this PR , I am not an expert in Perl , let me know if it does not break the test ( I have already test it , but an extra eyes on this might be good )

**Comment by zerodefect** [comment]
Yes, I'll try it out by the end of the day tomorrow. Happy to help.

File: README.md [hunk 1]
```diff
@@ -76,6 +76,17 @@ The command generates the following example.c.patch patch:
      {
          int i,x;
 ```
+
+If you want to use a differnt target for your code you can create a file targets.conf
```
Comment (StefanHamminga) [inline README.md]:
```suggestion
If you want to use a different target for your code you can create a file targets.conf
```

**Comment by zerodefect** [comment]
You beat me to it. I've just tried it this evening. It works :)

**Comment by VictorRodriguez** [comment]
Sorry @zerodefect my apologies , next time I will wait until your confirmation

**Comment by zerodefect** [comment]
I don't mind :) I thought you were waiting on me, so I didn't want you to think that I had forgotten about it.

**Pull request merged**